cloudant-client/src/main/java/com/cloudant/client/api/views/SettableViewParameters.java

```diff
@@ -231,6 +231,16 @@
          */
 
         RB update(String update);
+
+        /**
+         * A partition key can be specified when querying data so that results can be constrained
+         * to a specific database partition.
```

Comment:
Again we should add a comment here to make it clear this is only applicable to databases created with `partitioned:true` and link back.

Reply:
Is this okay?
```
        /**
         * A partition key can be specified when querying data so that results can be constrained
         * to a specific database partition.
         *
         * Only available in partitioned databases. To ensure a database is partitioned call
         * {@link Database#info()} and check that {@link DbInfo.Props#getPartitioned()} returns
         * {@code true}.
         *
...
```
I'll use the same snippet for search and query too.

Reply:
Sounds good, but I would change `ensure` to  either `check` or `verify`, since the only way to ensure it is partitioned is to create it that way.